WpfApp5/WpfApp5/ViewModel/ToysViewModel.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using WpfApp5.Model;
+
+namespace WpfApp5.ViewModel
+{
+    public class ToysViewModel : INotifyPropertyChanged
+    {
+        private ObservableCollection<Toy> toys;
+
+        public ObservableCollection<Toy> Toys
+        {
+            get
+            {
+                return this.toys;
+            }
+            set
+            {
+                if (this.toys == value)
+                    return;
+
+                this.toys = value;
+                this.OnPropertyChanged(nameof(this.Toys));
+            }
+        }
+
+        private Toy selectedToy;
+
+        public Toy SelectedToy
+        {
+            get
+            {
+                return this.selectedToy;
+            }
+            set
+            {
+                if (this.selectedToy == value)
+                    return;
+
+                this.selectedToy = value;
+                this.OnPropertyChanged(nameof(this.SelectedToy));
+            }
+        }
+
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        protected virtual void OnPropertyChanged(string propertyName)
+        {
+            if (this.PropertyChanged is null)
+                return;
+
+            this.PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+        }
+
+        public ToysViewModel()
+        {
+            this.Toys = new ObservableCollection<Toy>
+            {
+                new Toy{Brand = "Barbie", Model = "Doll", Price = 15.00m, Country = "USA", Age = 7, Type = "Fashion Doll", Image = "/Resourses/barbie.jpg"},
```

Comment:
Objects creation should be moved to a separate class with appropriate responsibility.

Reply:
-1 to Concerns separation.